index.html

```diff
@@ -19,7 +19,7 @@
         <!-- Navigation-->
         <nav class="navbar navbar-light bg-light static-top">
             <div class="container">
-                <a class="navbar-brand" href="#!">Start Bootstrap</a>
+                <a class="navbar-brand" href="#!">Dental Care</a>

```

Comment:
I'm not convinced about the heading change

Reply:
okay lets check for something else